rero_ils/modules/vendors/api.py

```diff
@@ -56,16 +57,24 @@ class Vendor(IlsRecord):
     def get_number_of_acq_orders(self):
         """Get number of acq orders."""
         from ..acq_orders.api import AcqOrdersSearch
```

Comment:
I think it's better to put this import on top.

Reply:
done